apps/web/src/pages/studio-onboarding/components/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import { hstack, vstack } from '@novu/novui/patterns';
+import { text } from '@novu/novui/recipes';
+import { styled } from '@novu/novui/jsx';
+import { Button, Tooltip } from '@novu/design-system';
+import { IconOutlineMenuBook } from '@novu/novui/icons';
+import { ROUTES } from '@novu/shared-web';
+import { useNavigate } from 'react-router-dom';
+import { css } from '@novu/novui/css';
+import { When } from '../../../components/utils/When';
+
+const Text = styled('a', text);
+
+export const Footer = ({
+  canSkipSetup = true,
+  learnMoreLink = '',
+  buttonText = 'Continue',
+  onClick,
+  loading = false,
+  tooltip = '',
+}: {
+  canSkipSetup?: boolean;
+  learnMoreLink?: string;
+  buttonText?: string;
+  onClick?: () => void;
+  loading?: boolean;
+  tooltip?: string;
+}) => {
+  const navigate = useNavigate();
+
+  return (
+    <div
+      className={css({
+        paddingTop: '50',
+        paddingBottom: '100',
+        backgroundColor: 'surface.panel',
+        position: 'fixed',
+        bottom: 0,
+        left: 0,
+        right: 0,
+      })}
+    >
+      <div className={vstack({ alignContent: 'center', height: '250' })}>
+        <div
+          className={hstack({
+            width: '880px',
```

Comment:
This is a temporary width until we have been able to build the correct look for the onboarding success page